composer.json

```diff
@@ -1,121 +1,125 @@
-        "laminas/laminas-json-server": "3.4.0",
-        "laminas/laminas-loader": "2.8.0",
-        "laminas/laminas-mvc": "3.3.0",
-        "laminas/laminas-mvc-console": "1.3.0",
-        "laminas/laminas-mvc-i18n": "1.3.0",
-        "laminas/laminas-servicemanager": "3.10.0",
-        "laminas/laminas-soap": "2.10.0",
-        "laminas/laminas-stdlib": "3.7.1",
-        "laminas/laminas-xmlrpc": "2.13.0",
-        "lcobucci/jwt": "4.1.5",
-        "league/omnipay": "3.2.1",
-        "mobiledetect/mobiledetectlib": "2.8.37",
-        "monolog/monolog": "2.3.5",
-        "mpdf/mpdf": "8.0.17",
-        "nyholm/psr7": "1.4.1",
-        "nyholm/psr7-server": "1.0.2",
-        "omnipay/stripe": "3.1.0",
-        "paragonie/multi-factor": "0.2.2",
-        "particle/validator": "2.3.4",
-        "pear/archive_tar": "1.4.14",
-        "phpmailer/phpmailer": "6.5.3",
-        "phpoffice/phpspreadsheet": "1.21.0",
-        "phpseclib/phpseclib": "2.0.35",
-        "psr/log": "1.1.4",
-        "ramsey/uuid": "4.2.3",
-        "rospdf/pdf-php": "0.12.63",
-        "smarty/smarty": "2.6.33",
-        "steverhoades/oauth2-openid-connect-server": "2.4.0",
-        "stripe/stripe-php": "7.112.0",
-        "symfony/config": "4.4.37",
-        "symfony/dependency-injection": "4.4.37",
-        "symfony/event-dispatcher": "4.4.37",
-        "symfony/event-dispatcher-contracts": "1.1.11",
-        "symfony/http-foundation": "4.4.37",
-        "symfony/yaml": "4.4.37",
-        "twig/twig": "3.3.8",
-        "vlucas/phpdotenv": "5.4.1",
-        "waryway/php-traits-library": "1.0.4",
-        "yubico/u2flib-server": "1.0.2"
-    },
-    "config": {
-        "platform": {
-            "php": "7.4"
-        }
-    },
-    "require-dev": {
-        "phpunit/phpunit": "9.*",
-        "symfony/panther": "1.*",
-        "zircote/swagger-php": "^3.2"
-    },
-    "repositories": [
-        {
-            "type": "vcs",
-            "url": "https://github.com/openemr/wkhtmltopdf-openemr"
-        },
-        {
-            "type": "git",
-            "url": "https://github.com/openemr/oe-module-cqm"
-        }
-    ],
-    "autoload": {
-        "exclude-from-classmap": [
-            "src/Common/Compatibility/Checker.php",
-            "library/classes/ClinicalTypes/",
-            "library/classes/rulesets/",
-            "library/classes/smtp/"
-        ],
-        "classmap": [
-            "library/classes"
-        ],
-        "files": [
-            "library/htmlspecialchars.inc.php",
-            "library/formdata.inc.php",
-            "library/sanitize.inc.php",
-            "library/formatting.inc.php",
-            "library/date_functions.php",
-            "library/validation/validate_core.php",
-            "library/translation.inc.php"
-        ],
-        "psr-4": {
-            "OpenEMR\\": "src"
-        }
-    },
-    "autoload-dev": {
-        "psr-4": {
-            "OpenEMR\\Tests\\": "tests\\Tests"
-        }
-    }
-}
+{

```

Comment:
there is something odd in this file that some sort of whitespace like change has happened on every line

Reply:
beats me. it's same as pulled from master unless phpstorm did something though all I did was add mustache and send to Ken in email...

Reply:
I had some weird formatting too after applying some of Jerry's and Stephen's changes... like the 'use' statements for the traits were WAY off the screen... there were like 100 tabs before each one. So weird. Styling looks ok now I think.

Reply:
I'll check this when I pull this down after merge. Bringing in. Ken you may want to start checking data population against measure. I'll be doing the same. Thanks